src/configurations.ts

```diff
@@ -116,7 +116,8 @@ const configurations = {
         isProduction: isProduction,
         logMissingConfigurations: readBooleanValue('LOG_MISSING_CONFIGURATIONS', true),
         failOnMissingConfigurations: readBooleanValue('FAIL_ON_MISSING_CONFIGURATIONS', isProduction),
-        autoDeleteTemp: readBooleanValue('AUTO_DELETE_TEMP_FILES', true)
+        autoDeleteTemp: readBooleanValue('AUTO_DELETE_TEMP_FILES', true),
+        baseDomain: readStringValue('BASE_DOMAIN', ''),
```

Comment:
Can we move this below isProduction?

I try to keep the order consistent between all the configurations for easy checking